src/laser/mod.rs

```diff
@@ -106,23 +76,23 @@ pub fn add_systems_to_dispatch(builder: &mut DispatcherBuilder<'static, 'static>
 		deps,
 	);
 	builder.add(
-		sampler::InitialiseLaserSamplerMasksSystem,
-		"initialise_laser_sampler_masks",
+		dipole_beam::AttachIndexToDipoleLightSystem,
```

Comment:
Looking at your changes, it's not clear to me where the existing additions of systems required for laser cooling went. Are they still part of the ECS dispatcher in this branch?